var/www/html/consul-templates/wp-config.php.ctmpl

```diff
@@ -71,13 +72,19 @@ define( 'WP_DEBUG_DISPLAY', {{key_or_default "wordpress/config/WP_DEBUG_DISPLAY"
  * Debug mode
  * Debugging? Enable these.
  */
-define( 'SAVEQUERIES', {{key_or_default "wordpress/config/SAVEQUERIES" "true" | parseInt}} );
-define( 'WP_DEBUG', {{key_or_default "wordpress/config/WP_DEBUG" "true" | parseInt}} );
+
+# turn on WP caching
+# TODO: template this with consul template to remove if no memcached servers are avaliable?
+# maybe just add it to the memcached-config.php file
+define('WP_CACHE', true);
+define( 'WP_APC_KEY_SALT', 'triton-001' );
+define( 'WP_CACHE_KEY_SALT', 'triton-001' );
```

Comment:
Do these sale definitions need to come from the config as well? Perhaps the can both share `{{env "CACHE_SALT"}}`?